endpoint/views/endpoint_view.xml

```diff
@@ -0,0 +1,163 @@
+                        </h2>
+                    </div>
+                    <notebook>
+                        <page name="main" string="Main">
+                            <group name="config1">
+                                <group name="main">
+                                    <field
+                                        name="company_id"
+                                        groups="base.group_multi_company"
+                                    />
+                                    <field name="route_group" />
+                                    <field name="exec_mode" />
+                                    <field
+                                        name="exec_as_user_id"
+                                        required="auth_type == 'public'"
+                                    />
+                                </group>
+                                <group name="auth" string="Auth">
+                                    <field name="auth_type" />
+                                </group>
+                            </group>
+                            <group name="config2">
+                                <group name="request" string="Request">
+                                    <field name="request_method" />
+                                    <field
+                                        name="request_content_type"
+                                        required="request_method in ('POST', 'PUT')"
+                                        invisible="request_method not in ('POST', 'PUT')"
+                                    />
+                                </group>
+                            </group>
+                        </page>
+                        <page name="code" string="Code" invisible="exec_mode != 'code'">
+                            <field name="code_snippet" widget="ace" />
+                        </page>
+                        <page
+                            name="code_help"
+                            string="Code Help"
+                            invisible="exec_mode != 'code'"
+                        >
+                            <field name="code_snippet_docs" />
+                        </page>
+                    </notebook>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="endpoint_endpoint_form_view">
+        <field name="name">endpoint.endpoint.form</field>
+        <field name="model">endpoint.endpoint</field>
+        <field name="inherit_id" ref="endpoint.endpoint_mixin_form_view" />
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <form position="attributes">
+                <!-- Needed just to make further inheritance happy -->
+                <attribute name="name">real</attribute>
+            </form>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="endpoint_endpoint_search_view">
```

Comment:
Recently I've added mixin views for search and list on v14. Could you please fwd port them? :pray: 

https://github.com/OCA/web-api/pull/87/commits/687032089b61d97d093af0d6ef0377a477d4f2c5